src/components/Game/Game.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useEffect, useState} from 'react';
+import _ from 'lodash'
+
+import './Game.scss';
+
+import FirstGameRow from "../FirstGameRow/FirstGameRow";
+import SecondGameRow from "../SecondGameRow/SecondGameRow";
+import ThirdGameRow from "../ThirdGameRow/ThirdGameRow";
+
+import {randomSum} from "../../services/math-utils";
+import {GameState} from "../../services/CommonTypes";
+
+const Game: React.FC = () => {
+    const MAX_NUMBER_OF_STARS = 9;
+    const MAX_NUMBER = 9
+    const MAX_NUMBER_OF_RETRIES = 5;
+    const GAME_TIME = 10
+    const TIMER_GAIN = 1
+    const [roundNumber, setRoundNumber] = useState(1);
+    const [availableNumbers, setAvailableNumbers] = useState(Array.from(Array(MAX_NUMBER), (_, i) => i + 1))
+    const [numberOfStars, setNumberOfStars] = useState<number>(randomSum(availableNumbers, MAX_NUMBER_OF_STARS))
+    const [selectedNumbers, setSelectedNumbers] = useState<Array<number>>([])
+    const [numberOfUsedRetries, setNumberOfRetries] = useState<number>(0);
+    const [gameState, setGameState] = useState<GameState>(GameState.NOT_INITIALIZED)
+    const [timer, setTimer] = useState<number>(GAME_TIME)
+
+    useEffect(() => {
```

Comment:
Effects can also be put into separate modules for readability, see https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook